libminifi/src/utils/StringUtils.cpp

```diff
@@ -510,4 +510,26 @@ bool StringUtils::matchesSequence(std::string_view str, const std::vector<std::s
   return true;
 }
 
+bool StringUtils::splitToUnitAndValue(std::string_view input, std::string& unit, int64_t& value) {
+  const char* begin = input.data();
+  char *end;
+  errno = 0;
+  value = std::strtoll(begin, &end, 0);
```

Comment:
Can we guarantee that strtoull will not throw?

Reply:
I naively just copied this from timeUtils but looking into it strtoull is guranteed to not throw, stoull is the one that throws this only sets the errno.
But using std::from_chars which doesnt rely on a global error variable is probably a better aproach, so I've refactored and added some tests for it in https://github.com/apache/nifi-minifi-cpp/pull/1543/commits/471e5dc9ff8473b04d9f801acd4cc9013dbb3db9